specificatie/genereervariant/openapi.yaml

```diff
@@ -245,7 +245,7 @@ paths:
           die je wil ontvangen geef je op met de resource-naam gevolgd door de property
           naam, met daartussen een punt. In de definitie van het antwoord kun je bij
           _embedded zien welke gerelateerde resources meegeleverd kunnen worden. Zie
-          [functionele specificaties](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/expand.feature).
+          [functionele specificaties](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.2.0/features/expand.feature).
```

Comment:
Moeten deze niet naar v1.3.0 verwijzen?

Reply:
Ja, dat klopt. Ik heb het even nagekeken. Dit staat nog verkeerd in de release 1.3.0 van de HC common yaml.
Als die wordt aangepast, dan kan ik opnieuw genereren, committen en psuhen.

Reply:
ik heb issue gemaakt in common om dit aan te passen. is m.i. voor nu niet iets wat deze pull request moet tegenhouden

Reply:
maar het is wel vreemd dat het eerst wel verwees naar v1.3.0

Reply:
Misschien begrijp ik het weer verkeerd hoor, maar wij genereren de plumbing code toch op de master op het moment van mergen van een pull request ? Waarom zit in deze PR dan allerlei javacode ?

Reply:
Zoals codegeneratie is ingericht, wordt code gegenereerd zodra er een genereervariant wordt gecommit. Dat gebeurde nooit voor de BAG omdat er een fout in de codegeneratie script zit. Ik kwam daar achter toen ik afgelopen maandag een melding van een afnemer kreeg dat de gegenereerde code die overeenkwam met de OAS specs. Zie: https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/commit/64d4888c387022e0a5e0bbe7bddaf61d275304fb.

Reply:
> ik heb issue gemaakt in common om dit aan te passen.

PR (https://github.com/VNG-Realisatie/Haal-Centraal-common/pull/121) staat klaar om gemerged te worden.